reference/api/documents.mdx

```diff
@@ -248,6 +248,45 @@ An array of documents. Each document is represented as a JSON object.
 ]
 ```
 
+#### `_vectors`
+
+`_vectors` is a special document attribute containing vector embeddings for AI-powered search. It accepts different types of values.
+
+`_vectors` may be an array of numbers:
```

Comment:
No, `_vectors` is always a map, whose keys are the name of embedders (or future embedders).

The value of each key can be an array or an object.